### ISSUE TITLE
1.1.4 checklist

### ISSUE THREAD

**Issue by bbolker** [glmmTMB/glmmTMB#820]
Since TMB 1.9.0 is out we should try to push out a new version soon. What should be on the list (release-critical, wishlist ...)?

**Comment by bbolker** [comment]
#823 makes this more time-sensitive.

## release-critical

- [ ] merge #819 
- update NEWS file as necessary
- reverse-dependency checking (@mebrooks ?)

## wish-list

- [ ] #814 
- [ ] #816 
- [ ] #803

**Comment by strengejacke** [comment]
Would be great to see #824 also included.

**Comment by bbolker** [comment]
I agree, but there's a little bit of feature/scope creep in that branch; in #828 I'm working on a bunch of cases having to do with handling mapped values correctly in confidence intervals (we want to have the option to include those rows in the output with `NA` values, to make it easier to match things up in `broom.mixed::tidy()` and so forth)

**Comment by mebrooks** [comment]
I ran checks on the master branch using R-hub. Debian Linux and Oracle Solaris were fine. Reverse dependencies were also fine. Of course I'll rerun them if we merge another branch.

Windows gave a NOTE that sounds like it's referring to [this line](https://github.com/glmmTMB/glmmTMB/blob/4edd2b573768ca56fdf0302093bc198607d6de46/glmmTMB/R/methods.R#L1032)

The check says 
```
#> map.match: no visible binding for global variable 'object'
[7958](https://builder.r-hub.io/status/glmmTMB_1.1.4.tar.gz-6dff321cae9342a181cbe9c870d9b10d#L7958)#> * checking R code for possible problems ... NOTE
[7959](https://builder.r-hub.io/status/glmmTMB_1.1.4.tar.gz-6dff321cae9342a181cbe9c870d9b10d#L7959)#> object
```

It also gave a NOTE about size, but haven't those been OK recently?
```
checking installed package size ... NOTE
  installed size is 26.9Mb
  sub-directories of 1Mb or more:
    test_data       2.2Mb
    libs           22.1Mb
    vignette_data   1.1Mb
```

**Comment by bbolker** [comment]
From [CRAN](https://cran.r-project.org/),

> CRAN submissions will be offline from Jul 22, 2022 to Aug 5, 2022

Can we aim to submit something next week (say) to avoid the rush?

**Comment by mebrooks** [comment]
Yes, I think I can submit it in the next few days if it's ready. On the checklist, all the critical items are done except to possibly add a NEWS item about changes to confint().

It sounds like #803 (branch `TMBad`) is finished, but I haven't tested it myself. @kaskr, do you think it's at a point where we should add it now?

What about #536  (branch `satterthwaite_df`)? @danielinteractive, how is that going?

**Comment by kaskr** [comment]
@mebrooks 

> do you think it's at a point where we should add it now?

I don't have strong opinions. It can wait until next submission if we want to give it more time for testing. I'll submit a PR now to trigger GA tests. Let's see what happens...

**Comment by bbolker** [comment]
I think we should skip #536, #803, and #816. It would be nice to get further on other tasks (#814, #828, #830, #831, #832), but I don't think we should let them stand in the way of submission.

**Comment by mebrooks** [comment]
The package (current master branch) passes checks on all 3 winbuilder platforms. I don't think anything has changed that would affect the reverse dependencies or other platforms.  So I'm willing to submit it to CRAN.

We can wait on the other updates until after submitting.

**Comment by bbolker** [comment]
Oops, I didn't see that you'd already successfully submitted to CRAN! Congratulations.  NEWS file is slightly out of date in this release, but I don't think that's a big deal/worth trying for a patch release ....

**Comment by mebrooks** [comment]
I'm glad it worked out anyhow :)
